csrc/executor_utils.cpp

```diff
@@ -916,7 +917,7 @@ void initializeCudaContext() {
   // lazily construct context if non-existing yet;
   // NOLINTNEXTLINE(cppcoreguidelines-init-variables)
   CUcontext pctx = nullptr;
-  AT_CUDA_DRIVER_CHECK(at::globalContext().getNVRTC().cuCtxGetCurrent(&pctx));
+  cuCtxGetCurrent(&pctx);
```

Comment:
This function (`initializeCudaContext`) doesn't seem to be used. There's calls to `at::cuda::jit::initializeCudaContext`, though.